lib/vmpooler/providers/base.rb

```diff
@@ -242,7 +242,7 @@ def vm_exists?(pool_name, vm_name)
         # returns
         #   nil when successful. Raises error when encountered
         def create_template_delta_disks(_pool)
-          raise("#{self.class.name} does not implement create_template_delta_disks")
+          puts("#{self.class.name} does not implement create_template_delta_disks")
         end
```

Comment:
should the delta disks stuff move to the vsphere provider?

Reply:
the actual implementation is in vsphere, but the manager does call it once per pool (and then marks it as done in redis). Not raising the error in the base class is what I have done for other vsphere specific method calls.

Another option would be to swallow the raised error or check that  the provider is vsphere before running it , but that would hide the base class ,ethod and I dont like it as much.

Reply:
Makes sense